go.mod

```diff
@@ -1,8 +1,11 @@
-module github.com/drone/go-login
+module github.com/sennotech/go-login
```

Comment:
That got to be reverted

Reply:
Sorry, this is an intermediate version of mine and I forgot to revert it. In addition, there is CI/CD in coding.net, so there is no need to integrate coding.net, and could you cancel the PR or need I cancel the PR...thank you for your reply.

Reply:
There is also ci/cd in Gitlab, but we still support it ;)